astrology.lic

```diff
@@ -276,15 +285,30 @@ class Astrology
     fix_standing
   end
 
-  def train_astrology
-    check_ripples
-    check_astral
-    check_weather
-    check_events(check_pools) if @predict_event
-    check_moons
-    check_heavens
+  def train_astrology(settings)
+    return unless settings
+    return unless settings.astrology_training
+
```

Comment:
Oh, will the transform in dependency make this a non-issue? Or should we output a deprecation warning in that case?